tests/TestCase/ToolbarServiceTest.php

```diff
@@ -73,6 +73,25 @@ public function testLoadPanels()
         $this->assertGreaterThan(1, $this->events->listeners('Controller.shutdown'));
         $this->assertInstanceOf('DebugKit\Panel\SqlLogPanel', $bar->panel('SqlLog'));
     }
+    
+    /**
+     * Test disabling panels.
+     *
+     * @return void
+     */
+    public function testDisablePanels()
+    {
+        $bar = new ToolbarService($this->events, ['panels' => [
+            'DebugKit.SqlLog' => false,
+            'DebugKit.Cache' => true,
+            'DebugKit.Session'
```

Comment:
Will this setup allow people to only disable the panels they want? What happens if a panel has both a numeric index and the same panel name is set to `false` as a key?